app/components/wds-avatar-stack.js

```diff
@@ -5,6 +5,7 @@ export default Component.extend({
 	classNames: ['wds-avatar-stack'],
 	maxStackSize: 5,
 	avatars: null,
+	translateFunc: null,
```

Comment:
as we talked
maybe we should treat i18next as peer dependency
as both mobile-wiki and discussions are using it